src/molecules/pagination/pagination.tsx

```diff
@@ -121,6 +115,19 @@ const Pagination: React.FC<PaginationProps> = (props) => {
 
   return (
     <PaginationWrapper className={cssClass('Pagination')} {...rest}>
+      {
+        total >= 3
```

Comment:
I think we should create constants to avoid using magic numbers, for example:

```
const MIN_PAGES_FOR_FIRST_PAGE_BUTTON = 3
const FIRST_PAGE = 1
```

Reply:
done